syncher/syncserver_test.go

```diff
@@ -0,0 +1,539 @@
+// Copyright (c) 2018 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package syncher
+
+import (
+	. "github.com/onsi/gomega"
+	"testing"
+
+	"github.com/projectcalico/app-policy/policystore"
+	"github.com/projectcalico/app-policy/proto"
+
+	envoyapi "github.com/envoyproxy/data-plane-api/api"
+)
+
+var addr1 = &envoyapi.Address{
+	Address: &envoyapi.Address_SocketAddress{SocketAddress: &envoyapi.SocketAddress{
+		Address:  "3.4.6.8",
```

Comment:
nit: use string constants; searching usage in the tests and correlating would be easier.

Reply:
done